aspnetcore/host-and-deploy/blazor/webassembly.md

```diff
@@ -17,14 +17,18 @@ By [Luke Latham](https://github.com/guardrex), [Rainer Stropek](https://www.time
 
 With the [Blazor WebAssembly hosting model](xref:blazor/hosting-models#blazor-webassembly):
 
-* The Blazor app, its dependencies, and the .NET runtime are downloaded to the browser.
+* The Blazor app, its dependencies, and the .NET runtime are downloaded to the browser in parallel.
 * The app is executed directly on the browser UI thread.
 
 The following deployment strategies are supported:
 
 * The Blazor app is served by an ASP.NET Core app. This strategy is covered in the [Hosted deployment with ASP.NET Core](#hosted-deployment-with-aspnet-core) section.
 * The Blazor app is placed on a static hosting web server or service, where .NET isn't used to serve the Blazor app. This strategy is covered in the [Standalone deployment](#standalone-deployment) section, which includes information on hosting a Blazor WebAssembly app as an IIS sub-app.
 
+## Brotli precompression
+
+When a Blazor WebAssembly app is published, the output is precompressed using the [Brotli compression algorithm](https://tools.ietf.org/html/rfc7932) at the highest level to reduce the app size and remove the need for runtime compression.
```

Comment:
We'll need to provide more guidance on how to take advantage of these precompressed files in different hosting environments (IIS, etc.) but for now this is fine.

Reply:
I'll open an issue to follow-up on it.